server/app/utilities/initialize_database.py

```diff
@@ -0,0 +1,30 @@
+"""Initialize database with pre-defined data."""
+
+from typing import Dict
+from app.models import Role
+from mongoengine import DoesNotExist
+
+default_roles = [
```

Comment:
This whole initialize database thing worked out great. I like the way it looks.

Reply:
Good, yeah I think it works well.